.gitignore

```diff
@@ -5,4 +5,5 @@ npm-debug.log
 coverage
 *~
 .idea
-.nyc_output
\ No newline at end of file
+.nyc_output
+package-lock.json
```

Comment:
package-lock.json needs to be committed to allow fixed versions, eg for a reconstructable CI run

Reply:
ok managed to produce builds with package-lock.json also by adding some missing dev dependencies, now package-lock.json is restored :+1: